aloha-core/src/main/scala/com/eharmony/aloha/audit/impl/scoreproto/ScoreAuditor.scala

```diff
@@ -0,0 +1,254 @@
+package com.eharmony.aloha.audit.impl.scoreproto
+
+import com.eharmony.aloha.audit.MorphableAuditor
+import com.eharmony.aloha.id.ModelIdentity
+import com.eharmony.aloha.reflect.RefInfo
+import com.eharmony.aloha.score.Scores.Score
+import com.eharmony.aloha.score.Scores.Score.BaseScore.ScoreType
+import com.eharmony.aloha.score.Scores.Score._
+
+import scala.collection.JavaConversions.{asJavaIterable, iterableAsScalaIterable}
+
+/**
+  * Created by ryan on 1/17/17.
+  */
+// TODO: Move to separate module.
+sealed abstract class ScoreAuditor[N] extends MorphableAuditor[Score, N, Score]
```

Comment:
I assume once we move away from protobufs, the Score protos will be replaced by some other type, and there would be some other ScoreAuditor type for non-protos.

Would it make sense to rename this to ScoreProtoAuditor instead?